contract_comm/currency/currency.go

```diff
@@ -221,17 +221,18 @@ func GetBalanceOf(accountOwner common.Address, contractAddress common.Address, g
 //-------------------------------
 func retrieveWhitelist(header *types.Header, state vm.StateDB) ([]common.Address, error) {
 	returnList := []common.Address{}
-	gasCurrencyWhiteListAddress, err := contract_comm.GetRegisteredAddress(params.GasCurrencyWhitelistRegistryId, nil, nil)
+
+	_, err := contract_comm.MakeStaticCall(params.GasCurrencyWhitelistRegistryId, getWhitelistFuncABI, "getWhitelist", []interface{}{}, &returnList, params.MaxGasForGetWhiteList, header, state)
 	if err != nil {
```

Comment:
You can remove lines 226-235, as all of this error logging will happen at the contract_comm level (see https://github.com/celo-org/celo-monorepo/pull/1292/files for this feature being enhanced)

Reply:
I will leave in for now as this change may go out before the other PR